### ISSUE TITLE
ReactDOM.render is no longer supported in React 18

### ISSUE THREAD

**Issue by alvarotrigo** [alvarotrigo/react-fullpage#441]
### Description

ReactDOM.render is no longer supported in React 18. 
The examples need to be updated to fix that.

**Comment by alvarotrigo** [comment]
[The examples](https://github.com/alvarotrigo/react-fullpage/blob/master/examples/react/src/index.js#L147C1-L148C39) use  the old syntaxt:

```js
const rootElement = document.getElementById('react-root');
ReactDOM.render(<App />, rootElement);
```

We probably need to update it for this instead? 

```js
import ReactDOM from 'react-dom/client';

const root = ReactDOM.createRoot(document.getElementById('react-root'));
root.render(<Fullpage />);
```

Would that be enough @jason-warner , @AbobkerElaghel, @ankitsinghaniyaz, @cmswalker?

**Comment by alvarotrigo** [comment]
Alright! I think it is enough :)
But if you have any other suggestions I'll be happy to add them too!

Merged into 0.1.46! 🥳
